bin/test-cleanup

```diff
@@ -22,6 +22,9 @@ echo "cleaning up jaeger extension resources, if present [${k8s_context}]"
 echo "cleaning up the all namespaces labelled with linkerd.io/is-test-data-plane"
 kubectl --context="$k8s_context" delete ns -l linkerd.io/is-test-data-plane
 
+echo "cleaning up ohter cluster-scoped resources labelled with linkerd.io/is-test-data-plane"
+kubectl --context="$k8s_context" delete clusterRole,clusterRoleBindings -l linkerd.io/is-test-data-planee
```

Comment:
`all` could not be used here, as its only meant for `namespace-scoped` resources like pods, etc. Are there any other cluster-scoped resources that we could add? 🤔